### ISSUE TITLE
TensorFlow Lite for Microcontrollers sigaborts with a MobileNetV2 alpha=0.1 model

### ISSUE THREAD

**Issue by dansitu** [tensorflow/tensorflow#41674]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): MacOS 10.15.5, Linux
- GCC/Compiler version (if compiling from source): Apple clang version 11.0.3 (clang-1103.0.32.62) c++1

**Describe the current behavior**

I am using TensorFlow Lite for Microcontrollers at commit 4f69f62c61ecf3cd23286324af62d00643186ec2.

I've trained two MobileNetV2 models in Keras with 48x48 input size and a single input channel, then converted to int8 quantized.

I am attempting to run both models using TensorFlow Lite for Microcontrollers on x86, built with clang on MacOS and with gcc on Ubuntu.

The first model has a MobileNetV2 filter scaling factor (_a_) of 0.35. This model runs perfectly.

The second model has a scaling factor of 0.1. This model sigaborts during the `Invoke()` call.

Strangely, both models run perfectly when executed from the OpenMV H7+ (Arm Cortex-M7), and the smaller model runs perfectly on the H7. It might be worth noting that on the OpenMV devices the model is stored in dynamic memory. That said, I've tried declaring the model without static on x86 and it has no impact.

I've attached zips containing both model files, plus an example program that exhibits the sigabort.

To build and run the example with an empty input:

```
make -f Makefile.tflite
./build/edge-impulse-standalone ""
```

Within the example code, the call to `Invoke()` is on line 260 of `edge-impulse-sdk/classifier/ei_run_classifier.h`.

To switch to the 0.35 model, which doesn't sigabort, replace `tflite-model`, `model-parameters`, and `edge-impulse-sdk` directories with the versions contained within `0.35 grayscale.zip`.

**Describe the expected behavior**

The _a_=0.1 model should run successfully on x86, the same as the 0.35 does.

**Standalone code to reproduce the issue**
Example code here:

[example-standalone-inferencing.zip](https://github.com/tensorflow/tensorflow/files/4968485/example-standalone-inferencing.zip)

The `.lite` model files are located here:
[models.zip](https://github.com/tensorflow/tensorflow/files/4968496/models.zip)

**Comment by janjongboom** [comment]
The actual crash is here: tensorflow/lite/kernels/internal/reference/integer_ops/add.h in the Add function.

```
TFLITE_DCHECK_LE(params.input1_offset, int8_max_value);
```

Here `params.input1_offset` is 128, which is invalid (max is 127).

**Comment by dansitu** [comment]
Thank you Jan! Since this seems to perhaps be a converter issue, I'm attaching the SavedModel files for both models.

[0.1-grayscale-savedmodel.zip](https://github.com/tensorflow/tensorflow/files/4973569/0.1-grayscale-savedmodel.zip)
[0.35-grayscale-savedmodel.zip](https://github.com/tensorflow/tensorflow/files/4973570/0.35-grayscale-savedmodel.zip)

**Comment by YarivCol** [comment]
@dansitu 
@janjongboom 
Hi, i have run into problems running similar NN to yours.
`base_model = tf.keras.applications.MobileNetV2(input_shape=(48, 48, 1), alpha=0.35, weights=None, include_top=False)

x = base_model.output

x = tf.keras.layers.Flatten()(x)

x = Dense(2)(x) #final layer with softmax activation for N classes

preds = tf.keras.layers.Softmax()(x)

model=Model(inputs=base_model.input,outputs=preds) #specify the inputs and outputs

# Converting a tf.Keras model to a TensorFlow Lite model.
converter = tf.lite.TFLiteConverter.from_keras_model(model)

# Convert the model to the TensorFlow Lite format with quantization
def representative_dataset():
  for i in range(500):
    yield([np.random.rand(1,48,48,1).astype(np.float32)])

# Set the optimization flag.
converter.optimizations = [tf.lite.Optimize.DEFAULT]
# Enforce full-int8 quantization (except inputs/outputs which are always float)
#converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
# Provide a representative dataset to ensure we quantize correctly.
#converter.inference_input_type = tf.int8  # or tf.uint8
#converter.inference_output_type = tf.int8  # or tf.uint8
converter.representative_dataset = representative_dataset

tflite_model = converter.convert()

open("mobilenet_test.tflite", "wb").write(tflite_model)
`
the model crush on the microcontroller in assert (tflite::PreprocessSoftmaxScaling)

thanks
